aws/rust-runtime/aws-config/Cargo.toml

```diff
@@ -26,6 +26,7 @@ aws-smithy-client = { path = "../../sdk/build/aws-sdk/sdk/aws-smithy-client", de
 aws-smithy-http = { path = "../../sdk/build/aws-sdk/sdk/aws-smithy-http" }
 aws-smithy-http-tower = { path = "../../sdk/build/aws-sdk/sdk/aws-smithy-http-tower" }
 aws-smithy-json = { path = "../../sdk/build/aws-sdk/sdk/aws-smithy-json" }
+aws-smithy-runtime-api = { path = "../../sdk/build/aws-sdk/sdk/aws-smithy-runtime-api" }
```

Comment:
I thought we said we weren't going to add this dep

Reply:
Hmm, if `aws-config` does not depend on `aws-smithy-runtime-api`, how does it know the type of `Interceptor` that's part of the method signature for `.interceptor` ?